leaf-ui/rappid-extensions/ElementInspector.js

```diff
@@ -641,43 +692,46 @@ var ElementInspector = Backbone.View.extend({
         }
 
         this.chart.display(context);
+        
     },
 
     getUDChartLabel: function(num) {
+        
         var res = ['0'];
         var curr = 'A'
         for (var i = 0; i < num; i++) {
             res.push(curr);
             curr = String.fromCharCode(curr.charCodeAt(0) + 1);
         }
         return res;
+    
     },
 
     /**
      * Updates the chart to represent data related to the the current user
      * defined function and satisfaction value(s)
      */
     updateChartUserDefined: function(event) {
+        
         var context = $("#chart").get(0).getContext("2d");
-        var numFuncSegments = this.intention.getNumOfFuncSegements();
+        var numFuncSegments = this.model.get('intention').getFuncSegments().length;
```

Comment:
This would never be null, right?

Reply:
Yup, at least one function segment should be in the list at this point. I'll add a comment